infra/scripts/delete-dbs.sh

```diff
@@ -6,22 +6,26 @@ aws rds delete-db-instance --db-instance-identifier robotshopmysql --skip-final-
 
 aws rds wait db-instance-deleted --db-instance-identifier robotshopmysql  --region ${AWS_REGION} --no-cli-pager
 
-aws rds delete-db-subnet-group --db-subnet-group-name robotshopmysql-subnet-group --region ${AWS_REGION} --no-cli-pager
+aws rds delete-db-subnet-group --db-subnet-group-name robotshop-mysql-subnet-group --region ${AWS_REGION} --no-cli-pager
```

Comment:
Let's not have db CRUD changes in this PR, these changes can be applied to the #14